components/needs_selector.js

```diff
@@ -23,12 +37,23 @@ const styles = theme => ({
     color: "white"
   },
   clearButton: {
-    textAlign: "right",
     textDecoration: "underline"
+  },
+  gridItemButton: {
+    textAlign: "center"
   }
 });
 
 class NeedsSelector extends Component {
```

Comment:
We should probably have a test that the new NeedsSelector and ProfileSelector behave appropriately under big / small widths.